cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -76,6 +76,40 @@ These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+#. Set the YARN Application Classpath.
+
+   .. highlight:: console
+
+   If you are running Spark programs, CDAP requires that additional entries be added to
+   the YARN application classpath, as the Spark installed on Cloudera Manager clusters
```

Comment:
as the Spark installed on Cloudera Manager clusters are hadoop-less build and does not include hadoop jars.